execute/exectypes/commit_data.go

```diff
@@ -0,0 +1,32 @@
+package exectypes
```

Comment:
I'm indifferent but why not just `types` and have an import alias to make it `exectypes`? I'm not crazy about it because it kinda makes imports harder (my IDE doesn't do auto-aliasing and there are a ton of packages called `types` already)

Reply:
Thats exactly why I wanted exec in the package name. There are so many `types` packages already and the aliases are often slightly different.